components/ui/scroll-area.tsx

```diff
@@ -18,7 +20,7 @@ const ScrollArea = React.forwardRef<
     <ScrollAreaPrimitive.Viewport className="h-full w-full rounded-[inherit]">
       {children}
     </ScrollAreaPrimitive.Viewport>
-    <ScrollBar />
+    {!disableScrollbar && <ScrollBar />}
```

Comment:
What's the use of this?

Reply:
needed to change its position in the dom, but I think I can get away with just exposing it's css

Reply:
Why are we trying to change its position again?

Reply:
dom postion was just an indication of where it will scroll. It will cover the title of the table if I wont change its css